src/screens/Settings/components/LanguageScreen/LanguageScreen.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Text, View } from 'react-native';
+import { useTranslation } from 'react-i18next';
+
+import { LanguageSkeleton } from './components/LanguageSkeleton';
+import { styles } from './styles';
+import { useLanguageScreenController } from './useLanguageScreenController';
+import { LanguageItem } from './components/LanguageItem';
+
+export function LanguageScreen(): JSX.Element {
+  const { language, onPressUA, onPressEN, isLoading } = useLanguageScreenController();
+  const { t } = useTranslation();
+
+  return (
+    <View style={styles.modalContainer}>
+      {isLoading && <LanguageSkeleton />}
```

Comment:
isLoading should not only render skeleton but also hide a layout which depends on this variable. You don't hide anything in here